Header header
geometry_msgs/Pose pose
hb_workspace_analysis/GraspPoint[] grasp
